langchain4j/src/main/java/dev/langchain4j/store/embedding/chroma/ChromaEmbeddingStore.java

```diff
@@ -0,0 +1,166 @@
+package dev.langchain4j.store.embedding.chroma;
+
+import dev.langchain4j.data.document.Metadata;
+import dev.langchain4j.data.embedding.Embedding;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.internal.Utils;
+import dev.langchain4j.store.embedding.EmbeddingMatch;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Collections.singletonList;
+import static java.util.stream.Collectors.toList;
+
+/**
+ * Represents a store for embeddings with Chroma backend.
+ * The cosine distance is always used as the distance method for Chroma implementation
+ */
+public class ChromaEmbeddingStore implements EmbeddingStore<TextSegment> {
+
+    private final ChromaClient chromaClient;
+    private final String collectionId;
+
+    /**
+     * Initializes a new instance of ChromaEmbeddingStore with the specified parameters.
+     *
+     * @param urlBase        The base URL of the Chroma service.
+     * @param timeout        The timeout duration for the Chroma client.
+     * @param collectionName The name of the collection in the Chroma service.
+     */
+    public ChromaEmbeddingStore(String urlBase, Duration timeout, String collectionName) {
+        this.chromaClient = new ChromaClient(urlBase, timeout);
+
+        Collection response = chromaClient.getCollection(collectionName);
+
+        collectionName = collectionName == null ? "default" : collectionName;
+
+        if (response == null) {
+            Collection collection = chromaClient.createCollection(new CollectionCreationRequest(collectionName));
+            collectionId = collection.id();
+        } else {
+            collectionId = response.id();
+        }
+    }
+
+    @Override
+    public String add(Embedding embedding) {
+        String id = Utils.randomUUID();
```

Comment:
minor: please use static imports when class name does not clarify or add relevant information

Reply:
done